app/assets/javascripts/admin/addon/controllers/admin-emojis.js

```diff
@@ -1,3 +1,10 @@
 import Controller from "@ember/controller";
+import { service } from "@ember/service";
 
-export default class AdminEmojisController extends Controller {}
+export default class AdminEmojisController extends Controller {
+  @service router;
+
+  get hideTabs() {
```

Comment:
I think don't worry about this for now, the whole header will be hidden not just the tabs. I will do this for all pages